contrib/ansible-role-php/vars/Debian.yml

```diff
@@ -1,38 +1,39 @@
 ---
+php_version: 5.6
 __php_packages:
-  - php5.6
-  - php5.6-mcrypt
-  - php5.6-cli
-  - php5.6-common
-  - php5.6-curl
-  - php-curl
-  - php5.6-dev
-  - php5.6-fpm
-  - php5.6-gd
+  - php{{ php_version }}
+  - php{{ php_version }}-mcrypt
```

Comment:
I guess because it was hacked many times in the past, and currently it doesn't look like a contrib role. But this is another story to discuss.